pkgs/top-level/aliases.nix

```diff
@@ -168,6 +170,7 @@ mapAliases ({
   google-gflags = gflags; # added 2019-07-25
   googleAuthenticator = google-authenticator; # added 2016-10-16
   grantlee5 = libsForQt5.grantlee;  # added 2015-12-19
+  graphite-web = throw "removed, unmaintained in nixpkgs"; # added 2020-03-11
```

Comment:
The error message should include that this is about `graphite-web`, same for `waitress-django`. Because otherwise people just get "error: removed, unmaintained in nixpkgs" which wouldn't be very informative :)

Reply:
Oops, yes, that would be better :) Fixed.